hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -564,6 +569,14 @@ public void submitRequest(OMRequest omRequest) {
     }
   }
 
+  public static boolean shouldIgnoreSnapshot(SnapshotInfo snapInfo,
+      SnapshotInfo.SnapshotStatus snapshotStatus,
+      boolean isSstFilteringServiceEnabled) {
+    return !snapshotStatus.equals(
```

Comment:
nit: use `==` for enum comparison to avoid null checks.

```suggestion
    return !snapshotStatus.equals(
        SnapshotInfo.SnapshotStatus.SNAPSHOT_DELETED)
```

Reply:
Done.